packages/ckeditor5-ckbox/lang/contexts.json

```
@@ -2,7 +2,7 @@
   "Open file manager": "A toolbar button tooltip for opening the file browser that allows inserting an image or a file to the editor.",
   "Cannot determine a category for the uploaded file.": "A message is displayed when CKEditor 5 cannot associate an image with any of the categories defined in CKBox while uploading an asset.",
   "Cannot access default workspace.": "A message is displayed when the user is not authorised to access the CKBox workspace configured as default one.",
-	"No permission for image editing. Try using the file manager or contact your administrator.": "The title of the notification displayed when there is no permission to edit assets.",
+	"You have no image editing permissions.": "The title of the notification displayed when there is no permission to edit assets.",
```

```suggestion
  "You have no image editing permissions.": "The title of the notification displayed when there is no permission to edit assets.",
```

Indentation unify (tabs vs spaces).